client/src/app/(onboarding)/login/page.tsx

```diff
@@ -1,3 +1,195 @@
+"use client";
+import "./login.css";
+import { Button, Typography, Divider, TextField, Link } from "@mui/material";
+import { useState } from "react";
+
 export default function Page() {
-  return <h1>Login Placeholder</h1>;
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const [error, setError] = useState(false);
+  const [emailHelperText, setEmailHelperText] = useState("");
```

Comment:
What is the email and password helperText use for?

Reply:
It is used to display what the user did wrong on inputing there email and password. I left it the way it is just incase in the future if there is certain password limitation we want to add. We would just have to update the if statements and set a new helper text for that problem